app/helpers/application_helper.rb

```diff
@@ -30,4 +30,15 @@ def current_action?(*args)
   def show_flash?
     alert != t('devise.failure.unauthenticated')
   end
+
+  def time_ago_with_tooltip(date, placement = 'top', html_class = 'time_ago')
+    capture_haml do
+      haml_tag :time, date.to_s,
+        class: html_class, datetime: date.getutc.iso8601, title: date.in_time_zone.stamp('Aug 21, 2011 9:23pm'),
```

Comment:
Line is too long. [112/80]<br>Align the parameters of a method call if they span more than one line.